tickets/.eslintrc.js

```diff
@@ -14,6 +14,14 @@ module.exports = {
   rules: {
     'react/prefer-stateless-function': [2],
     'react/forbid-prop-types': 2,
+    '@typescript-eslint/no-unused-vars': [
```

Comment:
Any reason why you add this eslint rule? Maybe better on another PR?

Reply:
This rule should already be covered by our eslint config (and `tsc` also catches it)